src/apps/integrations/router.py

```diff
@@ -41,3 +42,14 @@
         **AUTHENTICATION_ERROR_RESPONSES,
     },
 )(create_integration)
+
+router.get(
+    "/{applet_id}/{type}",
```

Comment:
Passing an invalid `type`, for example `loris` instead of `LORIS`, results in unhandled error.

At least for storing integrations we talked about using the general endpoint `/integrations` instead of using `type` as part of the url. Could that approach work here as well? Passing type as a parameter instead. Not necessary but it's more in line with existing conventions